ScriptingEngineTester/src/main/resources/zencode-tests/type-inference/implicit_constructors.zs

```diff
@@ -1,102 +1,117 @@
-#output: {"hello": ["world", 123, {"great": true}]}
+#output: {"hello": ["world", 123, {"great": true, "greater": "yes"}]}
```

Comment:
🎉 Works so far.  
I haven't tested how to do the `null` stuff, but well, for now we have file working!

Reply:
We can look into how to define implicit constructors for null...